bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
I think I missed the second component to your question - as for the log levels, you can use ```mock``` to mock the logger too, and then you can assert that the mock was called. After an object is mocked, you can inspect the mock object to see all calls to it and make your assertions. In the example I gave above, you can say things like ```Popen.mock_calls``` to see all the calls made to Popen and you can assert it was called with ```['/bin/echo', '"Hello World!"']``` and other nifty things. ```mock``` also has built in assertion helpers, like [```assert_called_with```](http://www.voidspace.org.uk/python/mock/mock.html#mock.Mock.assert_called_with). So I think you can use that technique to mock both the call to ```Popen``` and the calls to the log, and then you can assert that the correct log level was used.

Hope that helps!